cost-analyzer/templates/query-service-deployment-template.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if and (not .Values.agent) (not .Values.cloudAgent) (.Values.kubecostDeployment) }}
+{{- if and (.Values.kubecostDeployment.queryServiceReplicas) (.Values.kubecostDeployment.queryServiceReplicas.enabled) }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "query-service.fullname" . }}
+  namespace: {{ .Release.Name }}
+  labels:
+    {{ include "query-service.commonLabels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.kubecostDeployment.queryServiceReplicas.replicas | default 1 }}
```

Comment:
you don't need a default here. The values.yaml you set serves as the default

Reply:
I'm following [this example](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/templates/cost-analyzer-deployment-template.yaml#L17), which has a default even though it's defined in values.yaml.

Reply:
I've made some updates to the config, this default has been removed.